config/locales/es.yml

```diff
@@ -1823,12 +1823,12 @@ user_mailer.new_device_sign_in_attempts.events.sign_in_after_2fa: Autenticado
 user_mailer.new_device_sign_in_attempts.events.sign_in_before_2fa: Inicia sesión con contraseña
 user_mailer.new_device_sign_in_attempts.events.sign_in_unsuccessful_2fa: Error al autenticar
 user_mailer.new_device_sign_in_attempts.new_sign_in_from: Nuevo inicio de sesión potencialmente ubicado en %{location}
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.one: Su correo electrónico y su contraseña de %{app_name} se usaron para ingresar desde un nuevo dispositivo, pero <strong>la autenticación dio error</strong>.
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.other: Su correo electrónico y su contraseña de %{app_name} se usaron para ingresar desde un nuevo dispositivo, pero <strong>error al autenticar %{count} veces</strong>.
-user_mailer.new_device_sign_in_before_2fa.info_p1_html.zero: Su correo electrónico y su contraseña de %{app_name} se usaron para ingresar desde un nuevo dispositivo, pero <strong>la autenticación dio error</strong>.
+user_mailer.new_device_sign_in_before_2fa.info_p1_html.one: Su correo electrónico y contraseña de %{app_name} se usaron para iniciar sesión en un dispositivo nuevo, pero <strong>no se logró la autenticación</strong>.
```

Comment:
This translation works but I need to be able to distinguish this in Chinese translations. so it has the same styling.